src/handlers/psp.ts

```diff
@@ -213,121 +236,80 @@ const executeApiCalls = async (state: State): Promise<State> => {
   return { ...state, apiValuesBySubscriptionId };
 };
 
-const initializeProviders = async (state: State): Promise<State> => {
-  const { config, baseLogOptions } = state;
-
-  const airnodeWallet = ethers.Wallet.fromMnemonic(config.nodeSettings.airnodeWalletMnemonic);
-
-  const evmChains = config.chains.filter((chain) => chain.type === 'evm');
-  if (isEmpty(evmChains)) {
-    throw new Error('One or more evm compatible chains must be defined in the provided config');
+export const spawn = ({
+  providerSponsorSubscription,
+  baseLogOptions,
+  type,
+  stage,
+}: {
+  providerSponsorSubscription: ProviderSponsorSubscriptions;
+  baseLogOptions: utils.LogOptions;
+  type: 'local' | 'aws' | 'gcp';
+  stage: string;
+}) => {
+  // lambda.invoke is synchronous so we need to wrap this in a promise
+  switch (type) {
+    case 'local':
+      return new Promise((resolve, reject) => {
+        processSubscriptionsHandler({ providerSponsorSubscription, baseLogOptions }).then((data) => {
+          if (!data.ok) {
+            reject(data.message);
+          }
+          resolve(data.message);
+        });
+      });
```

Comment:
I was really concerned that we would not be able to run Airkeeper locally after this PR so I'm really happy to see that you are supporting local invocations 🙌🏻